### PR TITLE
Update last green commit from master, stable or main branch and retrieve rules_nodejs .bazelci from stable branch.

### PR DESCRIPTION
**Pull request by comius**
rules_nodejs stopped using master and switched to stable branch.
rules_pkg switched to main branch.

### CODE REVIEW

File: buildkite/bazelci.py [hunk 1]
```diff
@@ -2254,7 +2254,7 @@ def set_env_var(config_key, env_var_name):
     if not (
         is_pull_request()
         or use_but
-        or os.getenv("BUILDKITE_BRANCH") != "master"
+        or (os.getenv("BUILDKITE_BRANCH") != "master" and os.getenv("BUILDKITE_BRANCH") != "stable")
```
Comment (philwo) [inline buildkite/bazelci.py:2257]: While we're here, should be also add the check for `main`?

Comment (comius) [reply buildkite/bazelci.py:2257]: I'd rather wait for the first project to have it. This thing can produce problems if multiple branches are active.

Comment (philwo) [reply buildkite/bazelci.py:2257]: I see. Yeah, we have to be careful. Looks like rules_pkg has already switched to `main` - maybe that's a good enough reason?

Comment (comius) [reply buildkite/bazelci.py]: It is indeed. Done.

File: buildkite/bazelci.py [hunk 2]
```diff
@@ -2254,7 +2254,9 @@ def set_env_var(config_key, env_var_name):
     if not (
         is_pull_request()
         or use_but
-        or os.getenv("BUILDKITE_BRANCH") != "master"
+        or (os.getenv("BUILDKITE_BRANCH") != "master"
```
Comment (fweikert) [inline buildkite/bazelci.py:2257]: you can merge all three conditions:
... or os.getenv("BUILDKITE_BRANCH) not in ("master", "main", "stable") or ...

Comment (fweikert) [reply buildkite/bazelci.py:2257]: For clarity we should move it out of the first "not" clause, too.

File: buildkite/bazelci.py [hunk 3]
```diff
@@ -2254,9 +2254,7 @@ def set_env_var(config_key, env_var_name):
     if not (
         is_pull_request()
         or use_but
-        or (os.getenv("BUILDKITE_BRANCH") != "master" 
-            and os.getenv("BUILDKITE_BRANCH") != "stable"
-            and os.getenv("BUILDKITE_BRANCH") != "main")
+        or os.getenv("BUILDKITE_BRANCH") not in ("master", "stable", "main")
```
Comment (fweikert) [inline buildkite/bazelci.py:2257]: sorry for yet another late comment, but can you please get rid of the double negation?

Comment (comius) [reply buildkite/bazelci.py]: Done, applied de-Morgan laws :)

Comment (comius) [reply buildkite/bazelci.py]: Done on the comment as well.

Comment (fweikert) [reply buildkite/bazelci.py]: Nice, thanks!

**Review by fweikert — APPROVED** [review]

**Pull request merged**